tests/layer_tests/common/layer_utils.py

```diff
@@ -94,8 +95,18 @@ def fw_infer(self, input_data):
             # all input and output tensors have to be named
             assert out_obj.names, "Output tensor {} has no names".format(out_obj)
 
-            tensor_name = out_obj.get_any_name().split(':')[0]
-            result[tensor_name] = out_tensor
+            # For the new frontend we make this the right way because
+            # we know that tensor can have several names due to fusing
+            # and one of them the framework uses
+            if self.use_new_frontend:
+                for tensor_name in out_obj.get_names():
```

Comment:
Is it really needed to process every name from tensor? Is it not sufficient to process only `get_any_name`?

Reply:
`get_any_name()` can return any (not defined) tensor name from the list of possible tensor names (in case of fusing like for `Identity`) that can not match the single tensor name from the framework. So it is really needed